### ISSUE TITLE
XAML support

### ISSUE THREAD

**Issue by felipebaltazar** [Clancey/Reloadify3000#3]
Hi @Clancey !
Thanks for this awesome lib!
I've tried to implement it (based on MAUI pr 🙊) to use on Xamarin Forms projects.
But I got conflicts with visual studio HotReload (XAML) and CodeBehind (xaml) classes cant be reloaded because the XAML part is missing.
I don't know if I managed to express myself right here ...
But you cant check [this sample](https://github.com/felipebaltazar/Reloadify.Forms) if you run the platform project (Android or iOS) and try to do any change on `SecondPage.xaml.cs` you will get a `Exception` for `Activator.CreateInstance` on `HotReload.Views.cs` Line 118

I'm afraid a little bit that we can get this error from MAUI too, what do you think?

Thanks again!

**Comment by Clancey** [comment]
Yes, the Xaml code behind is not generated yet. We were talking about this during a team meeting the past week. It still needs to be solved :)
